### PR TITLE
Remove lookupId and uniqueId from frontend

### PR DESCRIPTION
**Pull request by nickscialli-usds**
## Related Issue or Background Info

- Issue #290 

## Changes Proposed

- Removes `lookupId`/`uniqueId` from UI and graphql queries

## Additional Information

- Didn't touch the backend to remove the `lookupId` altogether; based on the issue it sounds like we may have a use for `lookupId` eventually.

>Let's hide it from users until we encounter a valid use case.

### CODE REVIEW

File: frontend/src/app/patients/ManagePatients.tsx [hunk 1]
```diff
@@ -30,7 +29,6 @@ const patientQuery = gql`
 
 interface Patient {
   internalId: string;
-  lookupId: string;
```
Comment (dmethvin-gov) [inline frontend/src/app/patients/ManagePatients.tsx:33]: We may want to leave this in for now, assuming we aren't going to remove it from the backend too. At this point I don't know whether sites are using it or not, and whether we can look at live data to determine that. Originally the idea was that this field would be use to look up patients by room number or building or some such.

**Review by dmethvin-gov — COMMENTED** [review]
LGTM. Paging @TimBest on how much to slice out of the graphql side.

**Comment by nickscialli-usds** [comment]
> LGTM. Paging @TimBest on how much to slice out of the graphql side.

@timothybest-usds

**Comment by timothybest-usds** [comment]
> LGTM. Paging @TimBest on how much to slice out of the graphql side.

The plan is to leave `lookupId` alone on the graphql side but just stop showing it in the frontend for now. There is a possibility we will be adding this back in after some more user research

**Review by timothybest-usds — APPROVED** [review]

**Pull request merged**